### PR TITLE
Updated AndcultureCode.CSharp.Testing to 0.4.0 and Added the commente…

### LINKED ISSUE #30: Namespaces for Models are incorrect
For a couple of models, the namespaces are incorrect and do not reflect the folder structure.

Here is the list (<class> - <current namespace> - <proposed namespace>:
* `Acl` - `AndcultureCode.CSharp.Core.Models.Entities` - `AndcultureCode.CSharp.Core.Models.Entities.Acls`
* `Error` - `AndcultureCode.CSharp.Core.Models` - `AndcultureCode.CSharp.Core.Models.Errors`
* `Result<T>` - `AndcultureCode.CSharp.Core.Models` - `AndcultureCode.CSharp.Core.Models.Errors`

This is likely a breaking change, so we might need to think through how to best handle this.

### PR DESCRIPTION
**Pull request by SaidShah**
…d out tests

- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

Closes #30

### CODE REVIEW

**Comment by SaidShah** [comment]
@wintondeshong - All the tests are passing with the new updated AndcultureCode.CSharp.Testing package 👍 . Thank you.

File: test/AndcultureCode.CSharp.Core.Tests/Extensions/IAndcultureCodeWebHostBuilderExtensionsTest.cs [hunk 1]
```diff
@@ -1,71 +1,69 @@
...
-//            Environment.GetEnvironmentVariable(IAndcultureCodeWebHostBuilderExtensions.ASPNETCORE_ENVIRONMENT).ShouldBe(expected);
-//        }
-
-//        #endregion PreloadAmazonElasticBeanstalk
-//    }
-//}
\ No newline at end of file
+        #endregion PreloadAmazonElasticBeanstalk
+    }
+}
```
Comment (wintondeshong) [inline test/AndcultureCode.CSharp.Core.Tests/Extensions/IAndcultureCodeWebHostBuilderExtensionsTest.cs:69]: Nitpick: but could you add newlines to the end of all these files?

**Review by wintondeshong — CHANGES_REQUESTED** [review]
As you are traveling through, mind putting newlines on the files without any?

File: test/AndcultureCode.CSharp.Core.Tests/Extensions/IAndcultureCodeWebHostBuilderExtensionsTest.cs (hunk 1 above)
Comment (SaidShah) [reply test/AndcultureCode.CSharp.Core.Tests/Extensions/IAndcultureCodeWebHostBuilderExtensionsTest.cs]: @wintondeshong - Thanks for making me aware of this. I finished formatting the files. 👍

**Review by wintondeshong — APPROVED** [review]
Thanks @SaidShah !

**Pull request merged**